build/agent/Dockerfile

```diff
@@ -16,4 +16,7 @@ FROM loadimpact/k6:0.36.0
 WORKDIR /home/k6
 COPY --from=builder /go/k6 /usr/bin/k6
 COPY --from=builder /build/runner /bin/runner
+
+USER 1001
```

Comment:
I do not think this is necessary.
If you have a look at the official k6 base image, you will see that it uses a dedicated k6 user and not root.

```
USER 12345
```
According to /etc/passwd this corresponds to k6:x:12345:12345:12345:/home/k6:/bin/ash

Reply:
At least in /etc/passwd there also is no user with ID 1001. So you wold have to create it first.

Reply:
Good point - closing this one then. Thanks!